openerp/addons/base/tests/base_test.yml

```diff
@@ -92,56 +92,6 @@
         name: Child 2-1
         parent_id: test_categ_2
 -
-    2. Duplicate the parent category and verify that the children have been duplicated too and are below the new parent
```

Comment:
I still don't understand why this test should be removed. You should be able to duplicate a category; maybe the name has to be modified when copying.